java/client/src/test/java/glide/api/RedisClientTest.java

```diff
@@ -1681,6 +1687,113 @@ public void lpopCount_returns_success() {
         assertEquals(value, payload);
     }
 
+    @SneakyThrows
+    @Test
+    public void lpos() {
+        // setup
+        String[] set = {"a", "b", "c", "a", "c"};
+        String element = "b";
+        String[] args = new String[] {"set", "element"};
+        long index = 1L;
+
+        CompletableFuture<Long> testResponse = new CompletableFuture<>();
+        testResponse.complete(index);
+
+        // match on protobuf request
+        when(commandManager.<Long>submitNewCommand(eq(LPos), eq(args), any()))
+                .thenReturn(testResponse);
+
+        // exercise
+        CompletableFuture<Long> response = service.lpos("set", "element");
+        Long payload = response.get();
+
+        // verify
+        assertEquals(testResponse, response);
+        assertEquals(index, payload);
+    }
+
+    @SneakyThrows
+    @Test
+    public void lpos_withOptions() {
+        // setup
+        String[] set = {"a", "b", "c", "a", "c"};
+        String element = "b";
+        LPosOptions options = LPosOptions.builder().rank(1L).maxLength(1000L).build();
+        String[] args = new String[] {"set", "element", Arrays.toString(options.toArgs())};
```

Comment:
```suggestion
        String[] args = new String[] {"set", "element", "RANK", "1", "MAXLEN", "1000")};
```
Otherwise you're just comparing 
```
Arrays.toString(options.toArgs()) === Arrays.toString(options.toArgs())
```